feat: Adding Footer to the layout

titan-rocky opened the pull request:
Changes :
- Added a footer for the website
- replaced the dummy footer div in the main page component with the created footer
- added "Glacial Indifference" font for the bitspace logo in tailwind.config and globals.css

titan-rocky committed:
Footer

MKMukeshkannan commented on src/components/Footer.tsx:
```diff
@@ -0,0 +1,75 @@
+import Link from "next/link"
+import { useState } from "react"
```
- remove this un-used import

MKMukeshkannan reviewed (changes requested):
Follow the commit message convention.

titan-rocky committed:
fix: removed unused import

Changes:
- removed `import {useState} from 'react` in the footer page, which is unused

MKMukeshkannan commented:
Good Work 👍

titan-rocky commented:
Thank you ❤️
